src/popup/index.ts

```diff
@@ -237,9 +253,31 @@ document.addEventListener('DOMContentLoaded', () => {
 		});
 	}
 
+	function registerTab(tab: string, options: { buttonId?: string; scroll?: boolean } = {}) {
+		const button = document.getElementById(
+			options.buttonId ?? 'button-' + tab,
+		) as HTMLAnchorElement;
+
+		const buttonText = button.getElementsByTagName('span')[0] as HTMLElement;
+		buttonText.style.display = 'none';
+
+		const content = document.getElementById(tab) as HTMLDivElement;
+		content.style.display = 'none';
+		tabs[tab] = {
+			button,
+			content,
+			scroll: options.scroll ?? false,
+		};
+
+		button.addEventListener('click', () => selectTab(tab));
+		console.debug(logstr, 'registered tab', tab, tabs[tab]);
+	}
+
+	registerTab('general');
+	registerTab('verified');
+	registerTab('moderation');
+	registerTab('appearance');
 	popupPromise.then(items => selectTab(items.popupActiveTab));
```

Comment:
Make sure to migrate the stored tab value here, otherwise a user is going to get a tab-less popup the first time the load it after updating from the current live version

Reply:
how's it now?